mapper/orm/src/main/java/org/hibernate/search/mapper/orm/massindexing/monitor/impl/JMXRegistrar.java

```diff
@@ -0,0 +1,174 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.mapper.orm.massindexing.monitor.impl;
+
+import java.lang.invoke.MethodHandles;
+import java.lang.management.ManagementFactory;
+import java.util.concurrent.atomic.LongAdder;
+import javax.management.MBeanServer;
+import javax.management.MalformedObjectNameException;
+import javax.management.ObjectName;
+import javax.management.StandardMBean;
+
+import org.hibernate.search.mapper.orm.logging.impl.Log;
+import org.hibernate.search.mapper.orm.massindexing.monitor.IndexingProgressMonitorMBean;
+import org.hibernate.search.mapper.orm.massindexing.monitor.MassIndexerProgressMonitor;
+import org.hibernate.search.util.impl.common.LoggerFactory;
+
+/**
+ * Helper class to register JMX beans.
+ *
+ * @author Hardy Ferentschik
+ */
+public final class JMXRegistrar {
```

Comment:
This is completely out of scope, and this PR is taking long enough as it is. Adding back JMX support was never something we discussed.

Please remove this commit from the PR. You can keep it around somewhere if you want, but let's try to focus a bit?

Reply:
OK I will keep them in some patch files.